apps/console/app/lib/service.ts

```diff
@@ -46,6 +46,16 @@ export const kyFetch = ky.extend({
 
         if (typeof body === "object" && body && "message" in body)
           err.message = String(body.message);
+        // OpenAI Compatible error messages
+        else if (
+          typeof body === "object" &&
+          body &&
+          "error" in body &&
+          typeof body.error === "object" &&
+          body.error &&
+          "message" in body.error
+        )
+          err.message = String(body.error.message);
```

Comment:
```suggestion
    if (body && typeof body === "object") {
      if ("message" in body) {
        err.message = String(body.message);
      } else if (body.error?.message) {
        err.message = String(body.error.message);
      }
    }
```

Would you achieve the same with this?

Reply:
Yes, but then I can't correctly scope the comment to the specific block anymore. Performance-wise it is the same.

Reply:
Done